datafusion/core/benches/sql_planner.rs

```diff
@@ -288,10 +338,53 @@ fn criterion_benchmark(c: &mut Criterion) {
         })
     });
 
-    c.bench_function("logical_plan_tpcds_all", |b| {
+    // c.bench_function("logical_plan_tpcds_all", |b| {
+    //     b.iter(|| {
+    //         for sql in &all_tpcds_sql_queries {
+    //             logical_plan(&tpcds_ctx, sql)
+    //         }
+    //     })
+    // });
+
+    // -- clickbench --
+
+    let queries_file =
+        File::open(format!("{benchmarks_path}queries/clickbench/queries.sql")).unwrap();
+    let extended_file =
```

Comment:
I was confused at first what click bench Q48 was (as there are only 42 queries) -- but this now makes sense.


```
physical_plan_clickbench_q48
                        time:   [2.6437 ms 2.6674 ms 2.6943 ms]
Found 11 outliers among 100 measurements (11.00%)
  3 (3.00%) high mild
  8 (8.00%) high severe
```

It would probably be less confusing if this was called `physical_plan_clickbench_extended_q5` or whatever to align with the naming of suites